docs/content/docs/changelogs/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Working changelog
+draft: true
+description: Explanations of breaking and notable changes go here.  Images should live in docs/static/images.  The contents of this file are moved to a release notes page as parts of building docs for each release.
```

Comment:
unsure if we want to reuse this directory for all image resources or keep them inside this content bundle like under `docs/changelogs/images`

Reply:
Do we need images? I'd prefer to keep binaries out of the repo if possible. For the rest of the site, we have a CDN that we can upload to, but maybe that's too much overhead for this? What do you think?

Reply:
I'm not a fan of the cdn upload unless it's going to be incredibly easy to do.  If a dev is trying to add something here with an image then suddenly needs to go through some authentication process or ask someone else to do it, I think there's a good chance the image just gets skipped.

That said I'm not sure if we'll need images either, since the product isn't really some app with UI.  I'm fine advising against binary images like you say.  I'd also be fine with a temporary solution (like putting them in the repo) and revisiting later to see how big of a problem it is